packages/fxa-profile-server/scripts/test-ci.sh

```diff
@@ -2,12 +2,7 @@
 
 DIR=$(dirname "$0")
 
-sudo apt-get install -y graphicsmagick
-
-cd $DIR/../../fxa-shared
-npm ci
-
-cd ../fxa-profile-server
-cp ../version.json config
-npm ci
-npm test
+cp "$DIR/../../version.json" "$DIR/../config"
+yarn workspaces focus fxa-profile-server
+yarn workspace fxa-shared run build
```

Comment:
Some scripts use npx; do we want that here?

Reply:
I used npx when I can't be sure yarn is installed globally. On circleci it is, our docker images and teamcity it isn't.